src/configly/utilities.py

```diff
@@ -0,0 +1,29 @@
+def quote_string(value: str):
+    r"""Add syntactically-correct quotation characters to a string value.
+
+    >>> quote_string("foo")
+    '"foo"'
+
+    >>> quote_string("'bar'")
+    "'bar'"
+
+    >>> quote_string("ba'z")
+    '"ba\'z"'
+
+    >>> quote_string('''q"u'x''')
+    '\'q"u\'x\''
+    """
+    if value.startswith('"') and value.endswith('"'):
+        return value
+
+    if value.startswith("'") and value.endswith("'"):
+        return value
+
+    if '"' not in value:
+        return f'"{value}"'
+
+    if "'" not in value:
+        return f"'{value}'"
+
+    value = value.replace("'", "'")
```

Comment:
i dont understand need for this one

Reply:
good catch, this was supposed to be replacing `'` with an escaped ('\`'). I will correct it and the test.

Reply:
Black format is removing it for some bizarre reason 🤦